server/app/services/cloud/aws/FileDeletionFailureException.java

```diff
@@ -0,0 +1,8 @@
+package services.cloud.aws;
+
+/** Exception for when a request to delete a file from cloud storage has failed. */
+public final class FileDeletionFailureException extends Exception {
+  public FileDeletionFailureException(Exception originalException) {
+    super("The deletion was unable to be processed", originalException);
```

Comment:
The log statement in `AwsPublicStorage` will log the exception, so I think if I include a note about the job running monthly both in the exception and in the log statement itself it'll be logged twice? I don't have a preference about including the note in the exception itself vs in the log statement, let me know which you prefer!

Reply:
I meant add it to the string here instead of where you add the logger so that `e.toString()` will automatically include it. But maybe better to keep them separate in case we want to use `FileDeletionFailureException` elsewhere in the future. I just hadn't gotten to this file yet when I left that other comment 😂

Reply:
Oooh gotcha! Sounds good, will leave as-is!